src/components/atoms/TabLabel.tsx

```diff
@@ -4,26 +4,51 @@ import styled from 'styled-components';
 interface IProps {
   isActive: boolean;
   className?: string;
-  tabIndex: number;
-  onClick: (index: number) => void;
+  label: string;
+  onClick: (label: string) => void;
 }
 
-const LI = styled.li`
+const BUTTON = styled.button`
   display: inline-block;
   list-style: none;
-  margin-bottom: -1px;
-  padding: 0.5rem 0.75rem;
+  margin: 0px 4px;
+  padding: 0.5rem 0.75rem 0.75rem 0.75rem;
+  height 100%;
+  font-size: 0.9em;
+  vertical-align: middle;
 `;
 
 const TabLabel: React.FC<IProps> = props => {
-  const handleClick = () => props.onClick(props.tabIndex);
-  return <LI onClick={handleClick}>{props.children}</LI>;
+  const handleTabClick = () => {
+    props.onClick(props.label);
+  };
+  return (
+    <BUTTON
+      className={props.className}
+      onClick={handleTabClick}
+      role="tab"
+      aria-selected={props.isActive}
+      id={props.label}
+      aria-controls={`${props.label}-tab`}
+    >
+      {props.children}
+    </BUTTON>
+  );
 };
 
 export default styled(TabLabel)`
-  ${props =>
-    props.isActive &&
-    `border: solid #ccc;
-  background-color: white;
-  border-width: 1px 1px 0 1px;`};
+  background-color: ${props =>
+    props.isActive ? props.theme.palette.background.paper : '#eee'};
+  border: none;
+  border-top-left-radius: ${props => props.theme.shape.borderRadius};
+  border-top-right-radius: ${props => props.theme.shape.borderRadius};
+  border-bottom: ${props =>
+    !props.isActive ? 'solid 1.5px rgba(204,204,204, .5)' : 'none'};
+  box-shadow: 2px -2px 3px 0 #ccc;
+  font-weight: ${props => (props.isActive ? '700' : '')};
+  z-index: ${props => (props.isActive ? 301 : -300)};
+
+  :hover {
```

Comment:
Does this work? Thought you would need `&:hover`.

Reply:
Yes it does. Works with `&:hover` also :)